pipelines/nextflow/workflows/additional_seq_prepare/main.nf

```diff
@@ -35,33 +36,32 @@ Mandatory arguments:
   --production_name  Production name of the species
 
 Optional arguments:
-  --help             Show this help message and exit
```

Comment:
I don't think the helpMessage block here is needed at all no ?

Reply:
I thought I removed it, good catch :)